### DISCUSSION TITLE
为什么摇骰子会一直提醒我加入群组？我明明已经加入了啊？也操作了退出重新加入，没用...

### DISCUSSION THREAD

**Discussion by wdmzjxzl** [BurnHair/docs#141] [技术支持]
![82319fd1777cf49717da4dbde1cc964](https://github.com/BurnHair/docs/assets/36881585/b1107909-b76f-43cf-aa55-c8810ee02cc5)

**Comment by BennyThink** [comment]
因为你有重复注册，所有被检测到重复注册的用户均不可以丢骰子。

**Reply by wdmzjxzl** [reply]
不是重新换绑了tg，然后最新登录的账号也是兑换码登录的嘛？
